lnwallet/payment_descriptor.go

```diff
@@ -227,3 +227,52 @@ type paymentDescriptor struct {
 	// TLVs.
 	BlindingPoint lnwire.BlindingPointRecord
 }
+
+// asLogUpdate recovers the underlying LogUpdate from the paymentDescriptor.
```

Comment:
nit: `toLogUpdate`? or `toUpdateLog`? (wish we'd named it `UpdateLog` in the first place...)

Reply:
I can do `toLogUpdate`. I think keeping it named LogUpdate is best. When I think of an UpdateLog I think of a data structure with many updates. When I think of a LogUpdate I think of a single update that is found in that log.